public/app/features/explore/ExploreQueryInspector.tsx

```diff
@@ -57,15 +57,15 @@ function stripPropsFromResponse(response: any) {
   return clonedResponse;
 }
 
-interface Props {
+export interface Props {
```

Comment:
It's really just a nit (_and personal and totally opinionanted, so it's even less important than a nit 😄  )_, but I personally would avoid exporting anything that is not really needed outside of a module as I feel it adds noise.

In this case, `Props` is not really needed outside of here apart from the test, in which case you can do: 

```ts
import React, { ComponentProps } 'from 'react';
import { ExploreQueryInspector } from './ExploreQueryInspector';

type ExploreQueryInspectorProps = ComponentProps<typeof ExploreQueryInspector>;
```

what do you think?

Reply:
Uuu I like it and I agree! Will update this. Thanks!